portfolio/src/main/webapp/style.css

```diff
@@ -4,13 +4,23 @@
   width: 80%;
 }
 
+#about-me {
+  margin-left: auto;
+  margin-right: auto;
+  width: 80%;
+  margin-top: 30px;
+  margin-bottom: 50px;
```

Comment:
[Please sort css attributes alphabetically here and below](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order)